internal/planner/mapper/targetable.go

```diff
@@ -110,7 +110,7 @@ func filterObjectToMap(mapping *core.DocumentMapping, obj map[connor.FilterKey]a
 	for k, v := range obj {
 		switch keyType := k.(type) {
 		case *PropertyIndex:
-			subObj := v.(map[connor.FilterKey]any)
+			subObj, _ := v.(map[connor.FilterKey]any)
```

Comment:
todo: This looks like it was accidentally left in after debugging - can it be reverted?

Reply:
This line was causing a panic when the property value is nil. Is there a different way I should be handling the nil value here?

Reply:
Ah I see.  I would prefer a `if v != nil` wrapper then - it is a much stronger declaration of intent.

Reply:
done